src/EnergyPlus/StandardRatings.cc

```diff
@@ -1851,19 +1859,23 @@ namespace StandardRatings {
         static Real64 CoolingCapacityMax(0.0);           // cooling capacity of Mult-speed DX coil at max speed, [W]
         static Real64 CoolingElecPowerMax(0.0);          // outdoor unit electric power input at Max speed, [W]
         static Real64 PartLoadRatio(0.0);                // compressor cycling ratio between successive speeds, [-]
-        static Real64 PartLoadFraction(0.0);             // part-load fraction that account for the cyclic degradation, [-]
+        static Real64 PartLoadFactor(0.0);               // part-load factor that account for the cyclic degradation, [-]
```

Comment:
... from user-input PLF curve and C_D value.

Reply:
Done.